src/main/webapp/app/exam/manage/exams/exam-update.component.html

```diff
@@ -94,10 +95,14 @@ <h4>{{ 'artemisApp.examManagement.editExam' | artemisTranslate }}</h4>
                         labelName="{{ 'artemisApp.examManagement.examStudentReviewEnd' | artemisTranslate }}"
                         [(ngModel)]="exam.examStudentReviewEnd"
                         [error]="!isValidExamStudentReviewEnd"
+                        (valueChange)="validateReviewPhase()"
                         name="examStudentReviewEnd"
                         id="examStudentReviewEnd"
                     ></jhi-date-time-picker>
                 </div>
+                <div *ngIf="exam.reviewPhaseError" style="position: relative; left: 1250px" class="col-sm-5">
```

Comment:
a static offset is really not good here, since depending on window size the text will disappear. You should probably try to fix the text to the date box-
<details>
<img src=https://user-images.githubusercontent.com/33342534/128502251-f3cf3804-d662-4640-9555-142b177e6b5b.png>
<img src=https://user-images.githubusercontent.com/33342534/128502289-d86e1db6-7ece-47f6-a1d6-00ea5442a662.png>


</details>